src/localization.js

```diff
@@ -4,11 +4,62 @@ angular.module('ngLocalize')
             $html = angular.element(document.body).parent(),
             currentLocale,
             deferrences,
-            bundles,
-            cookieStore;
+            currentStorage,
+            bundles;
 
-        if (localeConf.persistSelection && $injector.has('$cookieStore')) {
-            cookieStore = $injector.get('$cookieStore');
+        function bind (fn, ctx) {
+            if (fn.bind) {
+                return fn.bind(ctx);
+            }
+
+            return function () {
+                return fn.apply(ctx, arguments);
+            };
+        }
+
+        function storage () {
```

Comment:
This would allow user to customize how the value is stored, so that the storage module is an optional dependency.